dockertest/output.py

```diff
@@ -737,11 +737,32 @@ def mustfail(cmdresult, failmsg=None):
         details = "%s\n%s" % (failmsg, cmdresult)
     if cmdresult is not None:
         OutputNotBad(cmdresult)
-    if cmdresult.exit_status == 0:
-        raise xceptions.DockerExecError("Unexpected zero exit code,"
-                                        " details: %s" % details)
+    # FIXME: temporary; remove once we no longer run on pre-1.10 docker
+    if not has_distinct_exit_codes():
+        expected_status = 1
+    if cmdresult.exit_status != expected_status:
+        raise xceptions.DockerExecError("Unexpected exit code %d; expected %d."
+                                        " Details: %s" % (cmdresult.exit_status,
+                                                          expected_status,
+                                                          details))
     return cmdresult
 
+HAS_DISTINCT_EXIT_CODES = None
+def has_distinct_exit_codes():
+    """
+    2016-03-23 **TEMPORARY** - for transition from docker-1.9 to 1.10
```

Comment:
My reasoning is that despite my best intentions, I may forget to remove this after docker-1.10. I want the next maintenance programmer to see this comment, read the datestamp, realize this code can be excised, and do so with minimal wasted time. Is there a better way to accomplish that?